docs/index.md

```diff
@@ -32,32 +34,28 @@ The code is kept here for easy maintenance.
 </p>
 <h3 align="center">Code scanning at ludicrous speed.<br />Find bugs and reachable dependency vulnerabilities in code.<br />Enforce your code standards on every commit.</h3>
 
-Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting dependency vulnerabilities, and enforcing code standards. [Get started →](/getting-started/quickstart)
-
-Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**.
-
-Its rules look like the code you already write; no abstract syntax trees, regex wrestling, or painful DSLs. Here's a rule for finding Python `print()` statements. Run it by clicking the [▸] button:
+Semgrep is an AppSec suite for finding bugs, detecting dependency vulnerabilities, and enforcing code standards. <!-- Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**. --> Its rules look like the code you already write -- no abstract syntax trees, regex wrestling, or painful DSLs. 
```

Comment:
```suggestion
Semgrep is an AppSec suite for finding bugs, detecting dependency vulnerabilities, and enforcing code standards. <!-- Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**. --> Its rules look like the code you already write -- no abstract syntax trees, regex wrestling, or painful DSLs. 
```
```suggestion
Semgrep is an AppSec suite for finding bugs, detecting dependency vulnerabilities, and enforcing code standards. <!-- Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**. --> Its rules look like the code you already write -- no abstract syntax trees, regex wrestling, or painful DSLs. 
```
```suggestion
Semgrep is an AppSec suite for finding bugs, detecting dependency vulnerabilities, and enforcing code standards. <!-- Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**. --> Its rules look like the code you already write -- no abstract syntax trees, regex wrestling, or painful DSLs. 
```

Why note delete the text that is commented out?

Reply:
I think I will delete the text if no one disagrees -- I commented it out as a temporary measure in case someone thinks I went too far and need to bring back some of the existing content